ui/narrative/methods/run_mash_dist_search/display.yaml

```diff
@@ -44,6 +44,11 @@ parameters :
             Maximum number of results to return
         short-hint : |
             Maximum number of results to return
+    cache :
```

Comment:
This should be an advanced parameter and should have docs saying that 99% of the time leaving this checked is the right option.

Reply:
I don't know that it makes sense to expose it in advanced either. We can leave it out of the display.yaml and then it could only be set by calling it in the code cell. Is there a situation that we'd want an end user to set this flag? I think only kbase testers would enable this.

Reply:
i have no strong opinion here

Reply:
happy to change it either way

Reply:
I'm fine with either way

Reply:
Leave it as advanced for now, Bill isn't sure if the parameter will get filtered out.